docroot/modules/custom/va_gov_backend/js/help_text_treatments.es6.js

```diff
@@ -0,0 +1,123 @@
+            .previousElementSibling.nodeName === "LABEL"
+        ) {
+          item.previousElementSibling.previousElementSibling.previousElementSibling.after(
+            item
+          );
+          item.style.margin = "0 0 10px 0px";
+        }
+      });
+    }
+  };
+
+  const alterHelpTextLocationOnLongTextFullHtmlField = (items) => {
+    if (items.length > 0) {
+      items.forEach((item) => {
+        if (
+          item.previousElementSibling !== null &&
+          item.previousElementSibling.previousElementSibling !== null &&
+          item.previousElementSibling.previousElementSibling
+            .firstElementChild !== null &&
+          item.previousElementSibling.previousElementSibling.firstElementChild
+            .nodeName === "LABEL"
+        ) {
+          item.previousElementSibling.previousElementSibling.firstElementChild.after(
+            item
+          );
+          item.style.margin = "0 0 10px 0px";
+        }
+      });
+    }
+  };
+
+  const alterHelpTextLocationOnParagraphsExperimentalWidget = (items) => {
+    if (items.length > 0) {
+      items.forEach((item) => {
+        if (
+          item.previousElementSibling !== null &&
+          item.previousElementSibling.firstElementChild !== null &&
+          item.previousElementSibling.firstElementChild.firstElementChild !==
+            null &&
+          item.previousElementSibling.firstElementChild.firstElementChild
+            .firstElementChild.firstElementChild !== null &&
+          item.previousElementSibling.firstElementChild.firstElementChild
+            .firstElementChild.firstElementChild.nodeName === "H4"
+        ) {
+          item.style.fontWeight = "400";
+          item.style.textTransform = "none";
+          item.style.margin = "0 0 10px 0px";
+          item.previousElementSibling.firstElementChild.firstElementChild.firstElementChild.firstElementChild.after(
+            item
+          );
+        }
+      });
+    }
+  };
+
+  /**
+   * Moves help text from below field / widget to below label.
+   * This approach relies on html structure patterns to broadly
+   * target fields and widgets. It is field name / type agnostic.
+   * The patterns below have comments that indicate their general scope.
+   */
+  Drupal.behaviors.vaGovtextContentDescriptionPlacement = {
+    attach(context) {
+      // Get help texts.
+      const textareaDescriptions = context.querySelectorAll(".description");
+      // Date fields.
+      alterHelpTextLocationOnDateField(textareaDescriptions);
+      // Text fields.
+      alterHelpTextLocationOnTextField(textareaDescriptions);
+      // Long text fields (No Wysiwyg).
+      alterHelpTextLocationOnLongTextPlainField(textareaDescriptions);
+      // Long text field full_html (Wysiwyg).
+      alterHelpTextLocationOnLongTextFullHtmlField(textareaDescriptions);
+      // Paragraphs experimental widget.
+      alterHelpTextLocationOnParagraphsExperimentalWidget(textareaDescriptions);
```

Comment:
```suggestion
      // Get help texts.
      const textareaDescriptions = context.querySelectorAll(".description");
      alterHelpTextLocationOnDateField(textareaDescriptions);
      alterHelpTextLocationOnTextField(textareaDescriptions);
      alterHelpTextLocationOnLongTextPlainField(textareaDescriptions);
      alterHelpTextLocationOnLongTextFullHtmlField(textareaDescriptions);
      alterHelpTextLocationOnParagraphsExperimentalWidget(textareaDescriptions);
```
With the good naming of the methods that you did, these comments are unnecessary.